ovs_performance.py

```diff
@@ -2485,7 +2494,7 @@ def get_ovs_version():
                                 die_on_error=True)
 
     m = re.search('.*([0-9]+.[0-9]+.[0-9]+).*',
-                  result.output)
+                  result.output.decode("utf-8", "ignore"))
     if m:
         return m.group(1)
```

Comment:
change this to return str(m.group(1)) or else the  StrictVersion(ovs_version) parts do not work. Please test this without the --no-bridge-config.